website/src/content/api-reference/commercetools-frontend-cypress.mdx

```diff
@@ -34,22 +34,100 @@ module.exports = (on, config) => {
 
 ```js title="cypress/support/commands.js"
 import '@commercetools-frontend/cypress/add-commands';
 ```
 
-## Commands
+# Commands
+
+## cy.loginToMerchantCenter
+
+This command performs the user login to the Merchant Center. It automatically detects the whether the application is running on localhost or on production and chooses the appropriate login mechanism.<br/>
+
+- When the application runs locally, the same mechanism used in the `cy.loginByOidc` is used.
+- When the application runs on production, a normal login flow is used where the user credentials are typed into the login form.
+
+### Usage
+
+```ts
+it('should render page 1', () => {
+  cy.loginToMerchantCenter({
+    entryPointUriPath: 'my-app',
+    initialRoute: `/${Cypress.env('PROJECT_KEY')}/my-app/page-1`,
+  });
+})
+it('should render page 2', () => {
+  cy.loginToMerchantCenter({ entryPointUriPath: 'my-app' });
+  cy.visit(`/${Cypress.env('PROJECT_KEY')}/my-app/page-2`)
+})
+```
+
+### Options
+
+Available options are:
+
+* `entryPointUriPath` (**required**): The application entry point URI path. This value is used to identify the correct application config.
+* `dotfiles` (optional): Pass a list of dotenv files that must be loaded when the `custom-application-config.json` is loaded (in case you are using environment placeholder). By default the following dotenv files are loaded: `.env` and `.env.local`. You can also define the values using paths relative to the application folder.
+* `initialRoute` (optional): If defined, the route is visited after login. If not defined, make sure to call `cy.visit` yourself.
+* `projectKey` (optional): The project key to access in the user session. The session token is valid for one project key at a time. Defaults to `Cypress.env('PROJECT_KEY')`.
+* `onBeforeLoad` (optional): Function called before the page has loaded all of its resources. Use this as a chance to interact for example with the browser storage.
+* `login` (optional): An object with the user login credentials `email` and `password`. If not provided, the `email` defaults to `Cypress.env('LOGIN_EMAIL') || Cypress.env('LOGIN_USER')` and the `password` defaults to `Cypress.env('LOGIN_PASSWORD')`.
+* `disableCacheAcrossSpecs` (optional): Turn off caching the session across specs. This is only relevant for Cypress version >= `10.9.0`.
+
+The command also requires to load the `custom-application-config.json` (automatically done via the Cypress task) and therefore it may need to load environment variables in case the application config uses environment placeholders.<br/>
+By default, the `.env` and `.env.local` files are attempted to be loaded from the application folder. You can pass a `dotfiles` option to pass a list of names/paths relative to the application folder in case the files in the project have a different name/location.
+
+<Info>
+
+The command also requires the following environment variables to be available: `PROJECT_KEY`, `LOGIN_USER`, `LOGIN_PASSWORD`.
```

Comment:
Since a user can provide the `login` option to this command, it seems `LOGIN_USER` AND `LOGIN_PASSWORD` env variables are not strictly required. Maybe they are needed internally for another thing?

Reply:
Yeah, let's rephrase that.

```suggestion
Make sure that the following environment variables are available in case the related options are not explicitly provided: `PROJECT_KEY`, `LOGIN_USER`, `LOGIN_PASSWORD`.
```

Reply:
@CarlosCortizasCT Something like that? ⬆️

Reply:
Yes, I find it more precise.
Thanks 🙇